src/python/libcgroup.pyx

```diff
@@ -615,6 +615,25 @@ cdef class Cgroup:
         if set_default:
             Cgroup.__set_default_systemd_cgroup()
 
+    @staticmethod
+    def clear_default_systemd_scope():
+        """Clear/Remove the default slice/scope from libcgroup
+
+        Description:
+        Delete the default slice/scope from the libcgroup var/run file.
+        Also delete the internal global variable within libcgroup so that
+        the default path is restored back to the root cgroup
+        """
+        try:
+            Cgroup.write_default_systemd_scope('', '')
+        except RuntimeError:
+            pass
+
+        try:
+            Cgroup.__set_default_systemd_cgroup()
```

Comment:
I guess `__set_default_systemd_cgroup()` is redundant here. The above `write_default_systemd_scope`, will call `__set_default_systemd_cgroup()` internally by default

Reply:
On a successful case, `__set_default_systemd_cgroup()` will get definitely get called by `write_default_systemd_scope()`, but I was afraid of the case where the call to `cgroup_write_systemd_default_cgroup()` failed and thus an exception was raised before the default was overwritten.  Is this crazy?

Reply:
Please consider this:
`write_default_systemd_scope()` `(cgroup_write_systemd_default_cgroup)` can fail at:
1. Fail to create `/var/run/libcgroup` directory or
2. Fail to open `/var/run/libcgroup/systemd` file for write or
3. Fail to write `slice/scope` into `/var/run/libcgroup/systemd`

In the above cases, the `default_systemd_scope()` if set before calling `clear_default_systemd_scope()` will remain set, and calling `Cgroup.__set_default_systemd_cgroup()` might set it again. Did I miss something here?

Reply:
Ooohhh... good counterpoint.  What if we do this?  `__set_default_systemd_cgroup()` will only get called once, but we guarantee it's always called.

```
        try:
            Cgroup.write_default_systemd_scope('', '', set_default=False) 
        except RuntimeError:
            pass

        try:
            Cgroup.__set_default_systemd_cgroup()
        except RuntimeError:
            pass

```

Reply:
This change looks good, the redundancy goes away.